### ISSUE TITLE
Clicking on the YouTube icon does not open YouTube

### ISSUE THREAD

**Issue by tecchan1107** [pnp/pnp.github.io#42]
Clicking on the YouTube icon does not open YouTube, but the community page.

![image](https://github.com/pnp/pnp.github.io/assets/51254139/b5bcdd68-87ed-47c5-b82b-114d82e5adf7)

**Comment by Adam-it** [comment]
yes this should be redirecting to `https://www.youtube.com/@MicrosoftCommunityLearning/videos` 
I will fix it up
